doc/UsersGuide/source/porting.rst

```diff
@@ -286,6 +286,93 @@ This construct is already accepted by some Modelica tools, but is not yet
 included in the current Modelica specification 3.5, nor even in the current working
 draft of 3.6, so it is not currently supported by OpenModelica.
 
+Incomplete specification of initial conditions
+------
+The simulation of Modelica models of dynamical systems requires the tool to
+determine a consistent initial solution for the simulation to start. In
+principle, this can be done by adding to the system equations one initial
+condition for each continuous state variable (after index reduction) and one
+initial condition for each discrete variable, then solving the resulting
```

Comment:
I think, this part should be rephrased.

Reply:
What's wrong with it? Isn't this what we are currently doing, in a nutshell?

Reply:
Yes, that is exactly what we do. I think the sentence is just difficult to read and a bit weird from a language point of view.

> In principle, this can be done by adding to the system equations one initial condition for each continuous state variable (after index reduction) and one initial condition for each discrete variable, then solving the resulting initialization problem.

Reply:
I guess you find it weird because it's built following some Italian structure, which I find natural. As I find weird some sentences like "If a Real variable is declared with the prefix discrete it must in a simulation model be assigned in a when-clause, either by an assignment or an equation" which is an obvious backversion from German.

What about this:

> To do so, the system equations are augmented by adding one initial
condition for each continuous state variable (after index reduction) and one
initial condition for each discrete variable. Then, the augmented system is solved upon initialization.

Reply:
You are probably right, and we don't have to change the text towards my personal preferences. However, I think it's now easier to read =)

Reply:
As an academic, I'm fairly used to criticism to everything I write. Happens all the time with papers, and my first thought is always, like, here we go again with the nasty reviewer from hell 😄 

But eventually papers, books, source code and even instruction manuals end up in a better shape thanks to reviews 😅